src/vorta/borg/create.py

```diff
@@ -36,6 +36,8 @@ def process_result(self, result):
                 self.app.backup_progress_event.emit(self.tr('Backup finished with warnings. See logs for details.'))
             else:
                 self.app.backup_progress_event.emit(self.tr('Backup finished.'))
+                self.app.backup_progress_event.emit(self.tr(''))
+                self.app.backup_log_event.emit('', {})
```

Comment:
```suggestion
                self.app.backup_log_event.emit('', {})
```

I imagine that to do the trick. Is there any log event emitted with vital information?

Reply:
> Is there any log event emitted with vital information?

?

Reply:
no